src/components/BaseHead.astro

```diff
@@ -1,17 +1,15 @@
 ---
-// Import the global.css file here so that it is included on
-// all pages through the use of the <BaseHead /> component.
-import '../styles/global.css';
+import "../styles/global.css";
```

Comment:
@dreyfus92 did you add rules somewhere that enforces double-quotes?

Reply:
Astro's Lang support extension for VSCode comes with Formatting powered by prettier so I tend to use astro's code formatter. Here's the link of the extension https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode